gradle-jdks/build.gradle

```diff
@@ -75,10 +76,10 @@ tasks.withType(JavaCompile).configureEach {
     it.options.errorprone.disable 'StrictUnusedVariable'
 }
 
-tasks.test {
-    environment("PROJECT_VERSION", project.version)
+tasks.withType(Test) {
+    environment(Map.of("PROJECT_VERSION", project.version))
     systemProperty 'ignoreDeprecations', 'true'
-    dependsOn project(':gradle-jdks-settings').tasks.named('build')
+    dependsOn project(':gradle-jdks-settings').tasks.withType(PluginUnderTestMetadata.class)
```

Comment:
before this change whenever we would run the tests in `gradle-jdks`, we would run the `gradle-jdks-settings` tests first